services/backend/src/service/basin_subzone.js

```diff
@@ -67,14 +67,14 @@ module.exports = (basinSubzonePersistence, seService) => {
      * @param {String} subzoneId subzone id
      * @param {String} seType strategic ecosystem type
      */
-    getCoverageInSE: async (subzoneId, seType) => {
+    /* getCoverageInSE: async (subzoneId, seType) => {
```

Comment:
In service there are also commented lines.

Reply:
Done!